src/interfaces/stylesheet/__init__.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from ..util import load_path
+
+def read_stylesheet(file_name):
+    path = load_path(os.getcwd() + "/interfaces/stylesheet/", file_name)
+    return open(path).read() if path else None
```

Comment:
Note: this doesn't explicitly close the file which could cause problems. It's probably a better idea to use 
```python
with open(...) as whatever: 
    ret = whatever.read()
    ...

return ret
```

Reply:
fixed